error.go

```diff
@@ -35,6 +35,17 @@ func (ind ErrInvalidNetworkDriver) Error() string {
 // BadRequest denotes the type of this error
 func (ind ErrInvalidNetworkDriver) BadRequest() {}
 
+// ErrNetworkNotSupportIsolation is returned if attempts to create an isolated
+// network, but the driver of that cannot guarantee no external connectivity.
+type ErrNetworkNotSupportIsolation string
```

Comment:
Let's not define new errors.
Please see https://github.com/docker/libnetwork/issues/269